tests/scripts/helpers/utils.py

```diff
@@ -770,8 +777,14 @@ def validate_local_monitoring_namespace(create_exp_namespace, list_reco_namespac
         terms_obj = list_reco_namespace["recommendations"]["data"][interval_end_time]["recommendation_terms"]
         current_config = list_reco_namespace["recommendations"]["data"][interval_end_time]["current"]
 
-        duration_terms = {'short_term': 4, 'medium_term': 7, 'long_term': 15}
-        for term in duration_terms.keys():
+        term_mapping = {"short": "short_term", "medium": "medium_term", "long": "long_term"}
+
+        if 'term_settings' in create_exp_recommendation_settings:
+            duration_terms = create_exp_recommendation_settings["term_settings"]["terms"]
+        else :
+            duration_terms = ['short', 'medium', 'long']
```

Comment:
Why are the values for each term removed here? Can you please ensure all the tests are running fine.

Reply:
This are not removed but will be taken from what terms user provides as input. If the terms are present it will pick that up else it will go ahead with all the terms.